scripts/create-clusters.sh

```diff
@@ -38,12 +38,13 @@ function create-insecure-registry() {
 function configure-insecure-registry() {
   local err=
   if sudo test -f "${docker_daemon_config}"; then
-    echo <<EOF "Error: ${docker_daemon_config} exists and \
-CONFIGURE_INSECURE_REGISTRY=${CONFIGURE_INSECURE_REGISTRY}. This script needs \
-to add an 'insecure-registries' entry with host '${CONTAINER_REGISTRY_HOST}' to \
-${docker_daemon_config}. Please make the necessary changes or backup and try again."
+    if sudo grep -q 'insecure-registries' ${docker_daemon_config}; then
+      echo <<EOF "Error: ${docker_daemon_config} exists and \
+is already configured with an 'insecure-registries' entry. \
+Please make sure it is removed and try again."
```

Comment:
(suggestion) Most probably the `insecure-registries entry` is added by our own script in local environment. In Such cases the entry could be correct. if so, just continue the work. otherwise report error about correcting insecure-registries entry.

Reply:
make sense. 

My fix is to check if the `insecure-registries` is set exactly by our own script or not.
If yes, just return success. If not, print error message.